ui/panels-plugin/src/components/OptionsEditorTabs/OptionsEditorTabs.test.tsx

```diff
@@ -72,16 +76,16 @@ describe('OptionsEditorTabs', () => {
 
   it('switches selected tab on click', () => {
     renderTabs();
-    const vizTab = screen.getByRole('tab', { name: 'Settings' });
-    userEvent.click(vizTab);
+    const jsonTab = screen.getByRole('tab', { name: 'JSON' });
```

Comment:
Switching which tab we test here, so that all the tabs get a little exercise in the tests. They keyboard-based test still opens the "Settings" tab.